src/Router/StorybooksRouter.tsx

```diff
@@ -22,6 +25,8 @@ export class StorybooksRouter extends React.Component<Props> {
         routes: this.props.routes,
         historyProtocol: "memory",
         initialRoute: this.props.initialRoute,
+        initialAppState: this.props.initialAppState,
+        initialState: this.props.initialState,
```

Comment:
Rather than passing some properties here and some to the returned ‘app’ component, pass them all here.